sig/sig-dataScience/charter.md

```diff
@@ -0,0 +1,44 @@
+# SIG Data Science Charter
+
+This charter adheres to the conventions described in the [Open Services Governance document](https://github.com/open-services-group/community/blob/main/governance.md) and uses the Roles and Organization Management outlined in [sig-governance](https://github.com/open-services-group/community/blob/main/committee-steering/governance/sig-governance.md).
+
+## Scope
+
+SIG Data Science is responsible for the development, exploration and communication of all data science and machine learning initiatives within the open services group. This includes cloud native machine learning tools and best practices, as well as AIOps tools to enhance the services run by the open services group.
```

Comment:
Does this also cover MLOps topics?

Reply:
I think that it should, yes.

I also think we may want to start to re-frame some of our language and start saying "MLops" instead of "Cloud Native Data Science" as I think there is a lot of overlap there and "MLOps" is a more established term. But open to holding off on that discussion till a later. 

Does that sound good? Should I explicitly add the term "MLOps" into thie paragraph somewhere? let me know :)

Reply:
I'd include MLOps #landgrab